presto-hudi/src/main/java/com/facebook/presto/hudi/HudiPageSourceProvider.java

```diff
@@ -116,6 +115,16 @@ public ConnectorPageSource createPageSource(
                     false);
         }
         else if (tableType == HudiTableType.MOR) {
+            // use first log file as base file for MOR table if it hasn't base file
```

Comment:
I find the use of the first log file as a base file, bit obtuse. Given the baseFile is an Optional anyway. Can we avoid this.

Reply:
because hudi check log-only  by basePath
https://github.com/apache/hudi/blob/aaddaf524a07daa1ffad6d6f5e04f6b130e5b069/hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/realtime/HoodieParquetRealtimeInputFormat.java#L69